beta/project-griffon/set-up-project-griffon.md

```diff
@@ -19,6 +19,25 @@ Click **Install** on the extension card. No extension settings are required.
 ## Add Project Griffon to your app
 
 {% tabs %}
+{% tab title="Android" %}
+1. Add the following libraries in your project's `build.gradle` file:
+`implementation 'com.adobe.marketing.mobile:core:1.4.0'`
```

Comment:
please list the libraries as a bulleted list:

Reply:
Done.